pkg/client/client.go

```diff
@@ -157,3 +157,20 @@ func (b *KafkaSourceBuilder) Sink(sink *duckv1.Destination) *KafkaSourceBuilder
 func (b *KafkaSourceBuilder) Build() *v1alpha1.KafkaSource {
 	return b.kafkaSource
 }
+
+// CloudEventOverrides adds given Cloud Event override extensions map to source spec
+func (b *KafkaSourceBuilder) CloudEventOverrides(ceo map[string]string) *KafkaSourceBuilder {
```

Comment:
Similar add/remove logic should be supported as for other sources. https://github.com/knative/client/blob/2ad8dfe483ffb5687779258e1e3a826ab7cb4225/pkg/sources/v1alpha2/apiserver_client.go#L195-L213